src/common/FaceliftUtils.h

```diff
@@ -219,4 +219,14 @@ class MostRecentlyUsedCache
     std::unordered_map<Key, decltype(m_list.begin())> m_map;
 };
 
+#ifdef FaceliftCommonLib_EXPORTS
```

Comment:
Please have a look at the way this is done in other files (such as "ipc-dbus.h") and do the same here, so that we keep consistency.